js/src/ScaledImageTexture.js

```diff
@@ -14,6 +14,11 @@ export default class ScaledImageTexture extends lng.textures.ImageTexture {
         }
     }
 
+    _getLookupId() {
+        const opts = this._scalingOptions;
+        return [this._src, opts['type'], opts['width'], opts['height']].join('-');
```

Comment:
Can we move this to:
`${src}-${opts.type}-${opts.width}-${opts.height}`

Seems to be 50% faster according to the benchmark
http://jsben.ch/YqBXQ